sklearn/linear_model/randomized_l1.py

```diff
@@ -307,7 +311,7 @@ class RandomizedLasso(BaseRandomizedLinearModel):
 
     See also
     --------
-    RandomizedLogisticRegression, LogisticRegression
+    RandomizedLogisticRegression, Lasso, ElasticNet
```

Comment:
My rationale is based on the data type that needs to be modeled. So methods for categorical and metric responses are kind of far away. This separation is maintained in the "See also" section of the lasso and logistic sides in `sklearn`, as they do not reference each other. I thought that it would be good to keep this separation. What do you think?